hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java

```diff
@@ -271,37 +251,6 @@ public void registerListener(Listener listener1) {
     listener = listener1;
   }
 
-  private static String convertTestUrls(
-      final String url,
-      final String fromNonSecureScheme,
-      final String fromSecureScheme,
-      final String fromDnsPrefix,
-      final String toNonSecureScheme,
-      final String toSecureScheme,
-      final String toDnsPrefix,
-      final boolean isAlwaysHttpsUsed) {
-    String data = null;
-    if (url.startsWith(fromNonSecureScheme + "://") && isAlwaysHttpsUsed) {
-      data = url.replace(fromNonSecureScheme + "://", toSecureScheme + "://");
-    } else if (url.startsWith(fromNonSecureScheme + "://")) {
-      data = url.replace(fromNonSecureScheme + "://", toNonSecureScheme + "://");
-    } else if (url.startsWith(fromSecureScheme + "://")) {
-      data = url.replace(fromSecureScheme + "://", toSecureScheme + "://");
-    }
-
-    if (data != null) {
-      data = data.replace("." + fromDnsPrefix + ".",
-          "." + toDnsPrefix + ".");
-    }
-    return data;
-  }
-
-  protected static String abfsUrlToWasbUrl(final String abfsUrl, final boolean isAlwaysHttpsUsed) {
-    return convertTestUrls(
-        abfsUrl, FileSystemUriSchemes.ABFS_SCHEME, FileSystemUriSchemes.ABFS_SECURE_SCHEME, FileSystemUriSchemes.ABFS_DNS_PREFIX,
-        FileSystemUriSchemes.WASB_SCHEME, FileSystemUriSchemes.WASB_SECURE_SCHEME, FileSystemUriSchemes.WASB_DNS_PREFIX, isAlwaysHttpsUsed);
-  }
-
```

Comment:
What will happen to https://github.com/ABFSDriver/AbfsHadoop/pull/5/files#diff-58f7a2a7f1c0b0f71e935687a17dedff0b9456c5aafddd69aff58df56f463973R200?

Reply:
Added back